docs/releases.md

```diff
@@ -40,12 +40,12 @@ Otherwise, we typically release every two weeks:
 
 ## Kubernetes Compatibility Matrix
 
-| Argo Workflows \ Kubernetes | 1.28 | 1.29 | 1.30 | 1.31 |
-|-----------------------------|------|------|------|------|
-| **main**                    | `✓`  | `✓`  | `✓`  | `✓`  |
-| **3.6**                    | `✓`  | `✓`  | `✓`  | `✓`  |
-| **3.5**                     | `✓`  | `✓`  | `✓`  | `?`  |
-| **3.4**                     | `?`  | `?`  | `?`  | `?`  |
+| Argo Workflows \ Kubernetes | 1.29 | 1.30 | 1.31 |
```

Comment:
I have problems with this table, but moved that to a slack discussion https://cloud-native.slack.com/archives/C0510EUH90V/p1732621744050569